syntax = "proto3";

import "google/protobuf/empty.proto";

option go_package = "gen/go/tus";

service TusService {
  rpc UploadTusData (TusUploadRequest) returns (TusUploadResponse);
  rpc CleanTusTable (google.protobuf.Empty) returns (TusCleanResponse);
}

message TusUploadRequest {
  string file_path = 1;
  int32 month = 2;
  int32 year = 3;
}

message TusUploadResponse {
  string status = 1;
}

message TusCleanResponse {
  string status = 1;
}
